hub/features/bbox.py

```diff
@@ -1,9 +1,22 @@
 from typing import Tuple
 
-import hub.features.feature_connector as feature_connector
+from hub.features.features import Tensor
 
-FeatureConnector = feature_connector.FeatureConnector
 
+class BBox(Tensor):
+    def __init__(self, xmin: float, ymin: float, 
```

Comment:
I don't think bbox needs its boundaries as input. This is metadata class not data class.

Reply:
There is a newer version